bin/plotting/pycbc_plot_singles_vs_params

```diff
@@ -128,8 +129,10 @@ if opts.z_var == 'density':
     norm = LogNorm()
     hb = ax.hexbin(x, y, norm=norm, vmin=1, **hexbin_style)
     fig.colorbar(hb, ticks=LogLocator(subs=range(10)))
-elif opts.z_var in sngl_statistic_dict:
+elif opts.z_var in ranking.sngls_ranking_function_dict:
     cb_style = {}
+    # FIXME: This seems wrong, we probably want the SingleDetTriggers class
+    #        to autocall into the ranking. Might try to fix.
```

Comment:
Given that FIXMEs make codeclimate angry, could you elaborate on what seems wrong here? This code just fetches whatever quantity the user wants to plot from the `SingleDetTriggers` instance. Is the idea that `z` should be fetched instead from some rank-related object which takes the `SingleDetTriggers` object as input?

Reply:
@titodalcanton If it's okay with you I'll leave this one to a separate PR (which I have ready, bar testing). The short answer is "yes" I would like `z` to come from a rank-related object, but fixing that involves a little more than a trivial change. (We want to avoid having to update this code if new sngl-det rankings are added).

.... The discussion here is already long, and I promise to have a proposal for removing this FIXME within a few days of this being merged.

Reply:
Sounds good.